requirements.dev.txt

```diff
@@ -27,8 +27,14 @@ pandas==0.25.3
 redis==3.5.3
 holidays==0.10.3
 sphinx
-recommonmark
+recommonmark~=0.6.0
 sphinx_rtd_theme
 jinja2
 flake8
 PuLP=2.1
+streamlit==0.69.1
+altair==4.1.0
+deepdiff==5.0.2
```

Comment:
rm deepdiff

Reply:
Not quite sure about that, but why do we need to remove deepdiff here?